.github/workflows/publish.yml

```diff
@@ -51,6 +51,10 @@ jobs:
         uses: docker/metadata-action@v3
         with:
           images: ${{ env.REGISTRY }}/nasa-ammos/aerie-merlin
+      - name: Scan container
+        uses: Azure/container-scan@v0
+        with:
+          image-name: ${{ env.REGISTRY }}/nasa-ammos/aerie-merlin
```

Comment:
Is this the correct order? I think we want to scan the image after we build it.
Also since we are using the `docker/build-push-action` which auto-labels the image with the proper tags, I think we might need to specify the tag in the `Azure/container-scan` step.

Reply:
Yes this makes sense. I wasn't clear what metadata action was doing. I'll switch it around.